SPARQLWrapper/sparql_dataframe.py

```diff
@@ -1,50 +1,69 @@
 """
 Query a SPARQL endpoint and return results as a Pandas dataframe.
 """
-from SPARQLWrapper import SPARQLWrapper2, SPARQLWrapper, CSV, SELECT
+import io
+from typing import Any, Dict, List, Union
+
+import rdflib.term
+
+from SPARQLWrapper.SmartWrapper import Bindings, SPARQLWrapper2, Value
+from SPARQLWrapper.Wrapper import CSV, SELECT, SPARQLWrapper
+
 
 class QueryException(Exception):
     pass
 
-def get_sparql_dataframe_orig(endpoint, query):
-    """ copy paste from: https://github.com/lawlesst/sparql-dataframe """
+
+def get_sparql_dataframe_orig(endpoint: str, query: Union[str, bytes]) -> Any: # -> pd.DataFrame:
```

Comment:
you can do:

```python
if TYPE_CHECKING:
    import pandas as pd
```

And then you can change this to:

```suggestion
def get_sparql_dataframe_orig(endpoint: str, query: Union[str, bytes]) -> "pd.DataFrame":
```

And you can do similar elsewhere.

Reply:
Some notes on this here: https://stackoverflow.com/questions/61384752/how-to-type-hint-with-an-optional-import

But maybe this is not the best idea, if we do this then pandas becomes a mandatory dependency for the purposes of type checking, so it may best to leave as is for now.

Marking as resolved.